### DISCUSSION TITLE
Donation / Collaboration

### DISCUSSION THREAD

**Discussion by gingerlime** [globaleaks/globaleaks-whistleblowing-software#3636] [General]
Thanks for the amazing work you and others are doing on GlobaLeaks. It's a fantastic platform.

I tried sending an email but it might have landed in spam, so hopefully I can reach you on here :)

I'm also an open-source developer, entrepreneur, and a privacy advocate with information security background. I collaborate with [NOYB](https://noyb.eu/en) on privacy issues relating to cookies and unique identifiers. I haven't contributed to GlobaLeaks (yet), but I'm hoping to do this in several ways. I wanted to suggest some opportunities that could benefit the project.

Very briefly, I'm looking into launching a new platform based on GlobaLeaks as a hosted service for the EU whistleblowing directive. I was thinking of pledging a certain portion of the profit as a donation to the open-source project, adding some code contributions as well (if they fit the project of course), and also to offer free GlobaLeaks hosting to non-profit organizations. I'm hoping to do it in a way that is ethical, no-strings-attached and would support the independence of this project.

Are you open to discuss some ideas?

**Comment by evilaliv3** [comment]
Thank you @gingerlime for reaching out and i'm sorry for the delay

Please feel free to reach us directly on slack at: community.globaleaks.org

We are happy to discuss a possible synergy and we will be back to you as well by email. I'm sorry for the delay in my answer.

**Comment by gingerlime** [comment]
No worries, @evilaliv3. Glad to hear back from you and I look forward to getting in touch. 👍
